### PR TITLE
[WA-14]: Forecast list 

### PR DESCRIPTION
**Pull request by Chakipaki**

### CODE REVIEW

File: weather-app/src/app/app.module.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {HeaderComponent} from "./header/header.component";
 import { HomeComponent } from './home/home.component';
 import { DetailsComponent } from './details/details.component';
 import { ForecastComponent } from './forecast/forecast.component';
+import {ForecastCardComponent} from "./forecast/component/forecast-card/forecast-card.component";
```
Comment (dmitry-chirva) [inline weather-app/src/app/app.module.ts:11]: That will be in forecast module

File: weather-app/src/app/forecast/component/forecast-card/forecast-card.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    maxTemp: "",
+    weather: "",
+    wind: "",
+    humidity: "",
+  };
+
+  constructor() { }
+
+  ngOnInit(): void {
+    console.log(this.dayState);
```
Comment (dmitry-chirva) [inline weather-app/src/app/forecast/component/forecast-card/forecast-card.component.ts:24]: remove console.log

File: weather-app/src/app/forecast/interfaces/forecastCard.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+interface forecastCardI {
```
Comment (dmitry-chirva) [inline weather-app/src/app/forecast/interfaces/forecastCard.ts:1]: export interface

File: weather-app/src/app/forecast/forecast.component.ts [hunk 4]
```diff
@@ -9,6 +10,100 @@ import { Router } from '@angular/router';
 export class ForecastComponent implements OnInit {
   constructor(private router: Router) {}
 
+  public city: string = "Kyiv, Ukraine";
+  public forecast: forecastCardI[] = [
```
Comment (dmitry-chirva) [inline weather-app/src/app/forecast/forecast.component.ts:14]: We don't need write public, typescript use public by default

File: weather-app/src/app/forecast/forecast.component.html [hunk 5]
```diff
@@ -1,3 +1,31 @@
-<h3>THIS IS FORECAST PAGE ⚡</h3>
-<button (click)="openHomePage()">to home</button> <span> | </span>
-<button (click)="openDetailsPage()">to details</button>
+<div class="forecast">
```
[the remark this answers is not in the document]
Comment (Chakipaki) [reply weather-app/src/app/forecast/forecast.component.html:1]: fixed

File: weather-app/src/app/forecast/component/forecast-card/forecast-card.component.ts (hunk 2 above)
Comment (Chakipaki) [reply weather-app/src/app/forecast/component/forecast-card/forecast-card.component.ts]: fixed

**Pull request merged**